### PR TITLE
FIX: Quantum bind_host must be 0.0.0.0 for non-HA envs.

### PR DESCRIPTION
**Pull request by xenolog**

### CODE REVIEW

**Comment by xenolog** [comment]
tested on centos/simple/quantum/gre

File: deployment/puppet/nova/manifests/network/quantum.pp [hunk 1]
```diff
@@ -18,7 +18,7 @@
     'DEFAULT/quantum_auth_strategy':     value => $quantum_auth_strategy;
     'DEFAULT/quantum_url':               value => $quantum_config['server']['api_url'];
     'DEFAULT/quantum_admin_tenant_name': value => $quantum_config['keystone']['admin_tenant_name'];
-    'DEFAULT/quantum_admin_username':    value => $quantum_config['keystone']['admin_username'];
+    'DEFAULT/quantum_admin_username':    value => $quantum_config['keystone']['admin_user'];
```
Comment (mihgen) [inline deployment/puppet/nova/manifests/network/quantum.pp:21]: Where all of these params go from?

**Comment by mihgen** [comment]
+1

**Comment by mihgen** [comment]
Tested in GRE mode. Connectivity between VM exists. Requires https://review.openstack.org/#/c/51376/ to work in full integration.

File: deployment/puppet/nova/manifests/network/quantum.pp (hunk 1 above)
Comment (xenolog) [inline deployment/puppet/nova/manifests/network/quantum.pp:21]: parser function sanitize_quantum_config() generate it.
Its get parameters, coming from nailgun ang glue its with pre-defined hardcored values.

File: deployment/puppet/l23network/manifests/l3/ifconfig.pp [hunk 2]
```diff
@@ -291,12 +291,13 @@
...
     refreshonly   => true,
   }
 
   # Ensure default route will be put in the right order
-  if defined(L23network::L3::Defaultroute[$def_gateway]) {
-    L3_if_downup <||> -> Defaultroute[$def_gateway]
-  }
+  # if defined(L23network::L3::Defaultroute[$def_gateway]) {
+  #   L3_if_downup <||> -> Defaultroute[$def_gateway]
+  # }
```
Comment (bogdando) [inline deployment/puppet/l23network/manifests/l3/ifconfig.pp:302]: -1
Would have broken fixes for PRD-1716 & RRD-2023

**Pull request merged**